assets/app/scripts/controllers/image.js

```diff
@@ -56,6 +60,51 @@ angular.module('openshiftConsole')
       );
     });
 
+    function updateTagsByName(){
```

Comment:
It might be nice to get this up into an ImageStream service & slim down the controller. 

```javascript
// ideally not have to pass the $scope into the service & just let it return the data
$scope.tagsByName = updateTagsByName(imageStream);
```